demisto_sdk/commands/validate/validation_results.py

```diff
@@ -43,9 +44,34 @@ def __init__(
         else:
             self.json_file_path = ""
 
+    def post_validation_results(
+        self,
+        config_file_content: ConfiguredValidations,
+        validation_results: List[ValidationResult],
+    ) -> Tuple[int, Set[str]]:
+        """Summarize the validation results and return the overall exit code and set of failing error codes.
+
+        Args:
+            config_file_content (ConfiguredValidations): The ConfiguredValidations object containing the warning level validations.
+            validation_results (List[ValidationResult]): The list of ValidationResult objects to summarize.
+
+        Returns:
+            Tuple[int, Set[str]]: The exit code and set of failing error codes.
+        """
+        exit_code = 0
+        failing_error_codes: Set[str] = set()
+        for result in validation_results:
+            if result.validator.error_code in config_file_content.warning:
+                logger.warning(f"<yellow>{result.format_readable_message}</yellow>")
```

Comment:
why are these only returned to logger? If calling by python code is this not returned?

Reply:
We return the exit code but we print out the errors.
This is how we designed it in the first place.